modules/mongodb/README.md

```diff
@@ -0,0 +1,342 @@
+<!--
+title: "MongoDB monitoring with Netdata"
+description: "Monitor the health and performance of MongoDB with zero configuration, per-second metric granularity, and interactive visualizations."
+custom_edit_url: https://github.com/netdata/go.d.plugin/edit/master/modules/mongodb/README.md
+sidebar_label: "MongoDB"
+-->
+
+# MongoDB monitoring with Netdata
+
+[`MongoDB`](https://www.mongodb.com/) MongoDB is a source-available cross-platform document-oriented database program.
+Classified as a NoSQL database program, MongoDB uses JSON-like documents with optional schemas. MongoDB is developed by
+MongoDB Inc. and licensed under the Server Side Public License (SSPL).
+
+source: [`Wikipedia`](https://en.wikipedia.org/wiki/MongoDB)
+
+---
+
+This module monitors one or more `MongoDB` instances, depending on your configuration.
+
+It collects information and statistics about the server executing the following commands:
+
+- [`serverStatus`](https://docs.mongodb.com/manual/reference/command/serverStatus/#mongodb-dbcommand-dbcmd.serverStatus)
+
+## Charts
+
+### Default chart
+
+Works with local and Atlas database servers
```

Comment:
done

Reply:
I am not 100% sure about the ping. It does send a command to the database and adds a round trip. I was actually thinking replacing the `collect` with `ping` in the check method. If we do just the connect as we do it now, check will fail anyway because it calls collect if the database is down or the connection uri is wrong. What do you think?